### DISCUSSION TITLE
Create fake-node in a real cluster that already exists

### DISCUSSION THREAD

**Discussion by KangYingjie0** [kubernetes-sigs/kwok#379] [Q&A] [answered]
Does kwok support that create fake-node in a real cluster, just like Kubemark?

**Comment by wzshiming** [comment, marked as answer]
Hope this is what you want

https://kwok.sigs.k8s.io/docs/user/kwok-in-cluster/

**Reply by KangYingjie0** [reply]
Thanks. I have followed the tutorial and practiced it. It works well. The project is so cool.

**Comment by KangYingjie0** [comment]
Hmm, I have another question. I find some versions in supported_releases.txt. But the version compatibility confuses me. Are they referring to the minimum or maximum version? For example, 1.25.3 is in the "supported_releases.txt" list, so can I use KWOK normally in a cluster with version=1.25.6 or version=1.25.0?

**Reply by wzshiming** [reply]
The last digit of the version in semver only means that there are bug fixes, you don't have to worry about compatibility issues, this list is of all the versions covered by the test and all of them work fine.

**Reply by KangYingjie0** [reply]
Thank you very much.
